skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/db/EditLetterHash.kt

```diff
@@ -17,8 +17,20 @@ fun Table.hashColumn(name: String): Column<ByteArray> =
 fun Column<ByteArray>.editLetterHash(): ValueClassWrapper<EditLetterHash, ByteArray> =
```

Comment:
Noe som ble tydelig for meg nå, er at hash-kolonnene i BrevredigeringTable og DocumentTable burde ha ansvar for innpakking i `EditLetterHash` slik at det er typen som eksponeres. Altså:
```
fun Table.hashColumn(name: String): Column<EditLetterHash> =
    binary(name).transform(columnTransformer({ it.hexBytes }, { EditLetterHash.fromBytes(it) }))
``` 
Dette er gitt at vi også modifiserer EditLetterHash med en getter `hex` som transformerer til ByteArray og `fromBytes` på companion.